locust/web.py

```diff
@@ -134,8 +134,11 @@ def index():
         @self.auth_required_if_enabled
         def swarm():
             assert request.method == "POST"
-            user_count = int(request.form["user_count"])
-            spawn_rate = float(request.form["spawn_rate"])
+            user_count = spawn_rate = 1
+            if request.form.get("user_count"):
+                user_count = int(request.form["user_count"])
+            if request.form.get("spawn_rate"):
+                spawn_rate = float(request.form["spawn_rate"])
```

Comment:
Instead of defaulting `user_count` and `spawn_rate` to `1` we should just return something like this (and ideally set the HTTP response code to `400`):
```javascript
{"success": False, "message": "You must specify user_count and spawn_rate"}
```

It would also be good to have a test for this (in `test_web.py`)

Reply:
Have commit the change of placeholders. Looks like this now
<img width="408" alt="image" src="https://user-images.githubusercontent.com/404966/102374184-0b74a880-3f8f-11eb-9c02-ec2edca602d5.png">

Reply:
@heyman I have added this response and removed the defaults values. Tests for this validations have been add to test_web.py
Also I have reorder the JS form submit code in order to check for response success in order to show the Locust Interface when tests begin to swarm, otherwise show the error message from the response data like:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/404966/102521269-b904bc00-4062-11eb-9ce3-e01849024cbc.png">

The error message in the image is shown because I have locally remove the required attributes from the form fields to test and show the error messages, but I keep it in the PR
Also I have added some UI improvements to prevent some clunky behavior of the dialogs that lead to this kind of weird visualization
<img width="487" alt="image" src="https://user-images.githubusercontent.com/404966/102523297-5660ef80-4065-11eb-953e-36920c9dea64.png">
That happens when you edit the test and without closing the edit form click on the stop button and then click on the new tests link that shows the start swarm form. So now when a dialog is about to open it close the other dialog if it's open